src/ts-mailgun.ts

```diff
@@ -203,6 +203,8 @@ export class NodeMailgun {
 			};
 
 			message = Object.assign(message, sendOptions);
+			
+			if (!body || sendOptions?.template) delete message.html;
```

Comment:
Please put `delete message.html` on the next line and wrap in curly braces to match the rest of the code, like this:

```typescript
if (!body || sendOptions?.template) {
	delete message.html;
}
```

Reply:
fixed!